Projects/Home/Sources/Home.swift

```diff
@@ -166,27 +188,19 @@ extension Home: Tabable {
 	}
 }
 
-extension GraphQL.HomeQuery.Data {
-	fileprivate var homeState: HomeState {
-		if isTerminated {
-			return .terminated
-		} else if isFuture {
-			return .future
-		} else {
-			return .active
-		}
-	}
-
-	private var isTerminated: Bool {
-		contracts.allSatisfy({ (contract) -> Bool in
-			contract.status.asActiveInFutureStatus != nil || contract.status.asTerminatedStatus != nil
-				|| contract.status.asTerminatedTodayStatus != nil
-		})
-	}
+extension UIRefreshControl {
+	func store<S: Store>(store: S, onReload: @escaping () -> S.Action, endLoadOn: S.Action...) -> Disposable {
```

Comment:
This became a bit hard to read I think. Also this extension should probably not be in Home.

```suggestion
	func store<S: Store>(_ store: S, send: @escaping () -> S.Action, endOn: S.Action...) -> Disposable {
```